README.md

```diff
@@ -50,7 +50,8 @@ Please send us a pull request with your suggestions.
 
 #### Doesn't this impact the right to free speech? Why are you stomping on my rights?
 
-See this XKCD for our feelings on this matter: https://xkcd.com/1357/
+In the many countries that do have the right to free speech, this means (with obvious exeptions) the government cannot arrest you for saying anything.  Many countries contain free speech in their constitution which can only apply to the government for enforcing or creating laws.
+As most companies and organizations are owned by private individuals, they are only restricted by laws that apply to people and their businesses.  The free speech sections contained in constitutions only apply to the government, restricting the kinds of laws that it can make.  So no, this does not impact your right to free speech.
```

Comment:
> I completely agree with you - it is usually irrelevant to software development per-se, however, this is not limited to Github, nor to communication directly related to development.

I've been viewing this as a CoC for Community Software Development. You're correct that my arguments are moot if we're talking about more than just that. 

> We could formulate this to something similar to "We respect and support freedom of speech. However, there is time and place for everything - before executing your right, think, is the pull request review really the place to debate political issues? Cake recipes? Favorite cartoon characters?"

I like this, rather than re-iterating what free speech is, it explains how "free speech" applies here. Rather than just "No free speech for you, we're not to government" it's, explains the relationship, 

My try:
"We respect and actively solicit your opinion and expertise in the context of this project, and want you to be able to freely articulate your ideas. While you're opinions outside of the scope of this project should be respected, if they do not relate to this project they are a distraction and if they hamper development by derailing conversations, cluttering issue trackers, or are being used to mock, harass or intimidate other developers they will not be welcome here. For persistent or egregious violations, we reserve the right to, as a community, remove you. If you are having your right to contribute to this project infringed upon by someone else in the community, please contact the abuse team explained in $abuse_team_section"

It covers the spirit, but reiterates things already mentioned elsewhere in the CoC. 

I'll let @CoWinkKeyDinkInc take another try at it. Their way with words has been great so far. :smiley: